app/next-client-app/types/dataset.ts

```diff
@@ -26,4 +26,15 @@ interface DataSetSRList {
   viewers: number[];
   admins: number[];
   editors: number[];
+  projects: number[];
+}
+interface Projects {
```

Comment:
```suggestion
interface Project {
```
This represents just one Project, not many.